src/eip7594/fft.c

```diff
@@ -130,7 +138,7 @@ C_KZG_RET fr_ifft(fr_t *out, const fr_t *in, size_t n, const KZGSettings *s) {
 
     fr_t inv_len;
     fr_from_uint64(&inv_len, n);
-    blst_fr_inverse(&inv_len, &inv_len);
+    blst_fr_eucl_inverse(&inv_len, &inv_len);
```

Comment:
Just noting here that `inv_len` is not a good name if this is the inverse of `n`. Why not `inv_n`?
Currently, it reads more like "the length of the inverse".